standard/expressions.md

```diff
@@ -680,23 +685,25 @@ When arguments are omitted from a function member with corresponding optional pa
 
 When a generic method is called without specifying type arguments, a ***type inference*** process attempts to infer type arguments for the call. The presence of type inference allows a more convenient syntax to be used for calling a generic method, and allows the programmer to avoid specifying redundant type information.
 
-> *Example*: Given the method declaration:
+> *Example*: Given the method declaration and use, it is possible to invoke the `Choose` method without explicitly specifying a type argument:
 >
 > ```csharp
 > class Chooser
 > {
->     static Random rand = new Random();
+>     static System.Random rand = new System.Random();
```

Comment:
I think it would be more idiomatic to have a `using System` directive.

Reply:
Examples as they existed *before* I proposed changes, are inconsistent; some have a `System.` prefix and some have a `using System;`.

Some observations:
1. Following our style, using the `using <namespace>;` approach adds 2 lines, that one plus a blank line.
2. In a few cases, we use a very long prefix, such as `System.Collections.Generic.`, which is a bit intrusive in code, and may be better suited as a `using <namespace>;`.

Reply:
removed name qualifiers